jjb/egeria/egeria-docker.yaml

```diff
@@ -9,36 +9,37 @@
     mvn-snapshot-id: logs
     container-public-registry: ''
     container-snapshot-registry: ''
+    container-staging-registry: ''
     build-timeout: 180
     stream:
       - 'master':
           branch: master
     jobs:
-      - github-maven-docker-verify:
+      - github-maven-docker-stage:
           project-name: egeria-configure
           mvn-goals: 'clean install'
```

Comment:
I don't think the maven-docker-stage job works with clean install goals. 
This job is designed to use clean deploy instead. 

Can we please get confirmation from the tech team that this is what they need? This job, compared to the verify, will generate staging artifacts into Nexus.

Reply:
@jwagantall, Actually in this docker jobs we are not using any artifacts and nexus. We are just creating a docker Image and pushing it to dockerhub.
The problem with the verify job is the triggers are set to execute for every PR request.
But here we want to execute on a particulate cron job or execute after the merge.
They have some maven jobs and this docker maven job is like a special case they wanted.

Reply:
In that case,, have you guys checked if the new docker jobs are better suitable for what you need?
https://github.com/lfit/releng-global-jjb/blob/master/jjb/lf-docker-jobs.yaml#L270
These new docker jobs are for repos that are 100% docker

Reply:
@jwagantall, We are using docker and maven.. 
These are the commands suggested by odpi team (Nigel) 
https://github.com/odpi/ci-management/issues/44#issuecomment-497432251

Reply:
Hi - I'm on the odpi team - and one of the authors of the docker/maven build

I originally updated the maven build to activate an optional profile triggered by a system env (-Ddocker)  to assist with consistency, & building and pushing using the spotify docker plugin. This was somewhat different to the 'normal' build which skips the docker part as it takes so long.

We're very open to changing that maven configuration based on recommendations as to lifecycle phase, plugins used, where to push to. The final destination should be docker hub, but if there is an option to push to a staging area first we can do that too.

Reply:
@jwagantall,  please can you Approve

Reply:
@planetf1
Thanks for the explanation. The merge job expects a "container-snapshot-registry" and the stage job a "container-staging-registry", but I believe in case of dockerhub the registry is the same in either case since there is no repos concept there 

Will approve this for now

Reply:
Just as a question ... what do other projects do? 
We are using the spotify docker plugin as - from the maven command line - it did what we need, but if there's a better way to build the image for our formal build, then that is an alternative too
As you indicate, should be ok for now though.